documentation/configuration.md

```diff
@@ -718,4 +744,4 @@ The following example shows the `Limits` portion of a collection rule that has t
         "ActionCountSlidingWindowDuration": "01:00:00",
     }
 }
-```
\ No newline at end of file
+```
```

Comment:
What changed here?

Reply:
I don't _think_ I changed anything there; I figured that was just from adding new lines in, so the text around it shifted (it's also highlighted in definitions.md).